### ISSUE TITLE
Identify invalid row from hasura error on bulk insert

### ISSUE THREAD

**Issue by kestutisg** [hasura/graphql-engine#4310]
I would like to be able to identify the row, that causes error on bulk insert (I'm inserting multiple objects of same type).

When inserting multiple rows to Postgres DB directly using SQL (INSERT INTO public."tableName" VALUES "row1", "row2";), I not only get an error, but also details like `DETAIL:  Failing row contains "row1"`. Using details information, at least in my case, it would be possible to identify problematic row.

**Comment by tirumaraiselvan** [comment]
Makes sense. Atleast for the `admin` role, this should be to easy to provide in the `extensions`.

The `run_sql` API already provides this info when it throws an error.

**Comment by tiagob** [comment]
We have a UI that allows multiple items to be created at once in a list. If one fails to insert into postgres (ex. a check constraint fails) we want to show an error next to that item instead of for everything. We ended up adding client side validation for this but it would be great if the server could send us back the right information.

**Comment by afourka** [comment]
thanks @kestutisg , @tiagob for reporting this issue.

I'am using the 2.0 version the index is well returned 

      "extensions": {
        "path": "$.selectionSet.insert_plot.args.objects[1]",
